src/usr/local/www/vpn_ipsec_phase2.php

```diff
@@ -595,7 +595,7 @@ function idinfo_to_pconfig($prefix, & $idinfo, & $pconfig) {
 	'*Protocol',
 	$pconfig['proto'],
 	$p2_protos
-))->setHelp('ESP is encryption, AH is authentication only.');
+))->setHelp('ESP (Encapsulating Security Payload) is encryption, AH (Authentication Header) is authentication only.');
```

Comment:
If you're writing both out, it's better to write out the full name and then put the abbreviation in parenthesis, e.g. "Encapsulating Security Payload (ESP)"